src/api.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const api = "https://news-ynvj.onrender.com/api";
```

Comment:
Nice work here! Take a look how we can make this even easier though by using axios.create (under 'Axios instance' a the bottom of this link below) 👍
https://notes.northcoders.com/courses/js-front-end/axios